### ISSUE TITLE
Maintenance

### ISSUE THREAD

**Issue by ghost** [tkashkin/GameHub#660]
Is this project still planned to be maintained or is it dead?

**Comment by PGSafarik** [comment]
I cannot and will not speak for the author. But I think he's probably got it pretty complicated right now. 

Anyway, I'm still using it myself, and I'm planning to make some modifications to it eventually - at least this way the project is not dead yet...

**Comment by sbritorodr** [comment]
Necromanzing an old thread to say if this program works good for you, GameHub crashes at downloading GOG games for me right now

**Comment by ghost** [comment]
That sucks. Thankfully there are plenty of GOG clients for Linux. Shame GameHub doesn't work

**Comment by sbritorodr** [comment]
Yeah, gamehub looks more robust than Heroic but definetely Heroic works much better nowadays. That's lame.

**Comment by patlefort** [comment]
I'd like to see a sign of life, it's my favored launcher but without support it will break and die. It's been over a year with 0 commit, is it time to move on?

**Comment by ghost** [comment]
Most likely.
